tests/backup_helper.go

```diff
@@ -8948,3 +8950,28 @@ func UpdateMaintenanceCronJob(backupLocation string, maintenanceJobType Maintena
 	_, err = k8sBatch.UpdateCronJob(cronJob)
 	return err
 }
+
+// DeleteAllDataExportCRs deletes the data export CR as soon as it is found in the given namespace
+func DeleteAllDataExportCRs(namespace string) error {
```

Comment:
I think its better have to filter options as well, in case if single namespace of multiple volumes and with same volume type and if we delete at namespace level the backup will be completely failed.

Reply:
@ak-px I thought about this, I will create another function called `DeleteDataExportCRs` which will take a map[namespace]pvcName
Can I do this in next PR or shall I add this function in the same PR?

Reply:
Currently this method won't block us as we are using the different vol type app. so it's ok to do in separate PR.